rest/rest-server/src/main/java/org/apache/brooklyn/rest/resources/EntityConfigResource.java

```diff
@@ -56,6 +58,11 @@
     @Override
     public List<EntityConfigSummary> list(final String application, final String entityToken) {
         final Entity entity = brooklyn().getEntity(application, entityToken);
+        if (!Entitlements.isEntitled(mgmt().getEntitlementManager(), Entitlements.SEE_ENTITY, entity)) {
+            throw WebResourceUtils.unauthorized("User '%s' is not authorized to see entity '%s'",
```

Comment:
we use `401 unauthorized` throughout.  but you are right, and we should fix in another PR -- https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.4.2 says `401 unauthorized` is actually for cases where a user has provided no authentication or wrong credentials.  (it should probably be called `unauthenticated` not `unauthorized`!)

best practise i think is:
* `404 not found` when we really should not be giving any info on the existence of a resource (in this case)
* `403 forbidden` -- when a user is able to see something but not act on it in the manner they are attempting